libraries/core_libs/network/src/tarcap/packets_handlers/pbft_sync_packet_handler.cpp

```diff
@@ -54,20 +47,12 @@ void PbftSyncPacketHandler::process(const PacketData &packet_data, const std::sh
     return;
   }
 
-  const size_t item_count = packet_data.rlp_.itemCount();
-
-  // No blocks received - syncing is complete
-  if (item_count == 0) {
-    pbftSyncComplete();
-    return;
-  }
-
   // Process received pbft blocks
-  bool last_block = packet_data.rlp_[0].toInt<bool>();
-
+  const bool pbft_chain_synced = packet_data.rlp_[0].toInt<bool>();
+  const bool last_block = packet_data.rlp_[1].toInt<bool>();
```

Comment:
pls add here comments with short explanation of what these 2 flags (pbft_chain_synced / last_block) mean and how they differ so other devs dont really need to go through code to understand it if they need to work with this part of code

Reply:
Fix adding comments